controller/src/test/resources/controller.config.properties

```diff
@@ -12,38 +12,38 @@
 # The commented out settings are not different than the hardcoded defaults in Config.java.
 
 
-controller.containerCount=4
```

Comment:
Now that you are addressing the configuration of Pravega, I ask the following: could we get rid of this file completely? I mean, I have always wondered why are there 2 separate files for configuration, one for the Segment Store and another for the Controller. Would it be better to have a single file with different "sections"? Maybe this is something to discuss with @shiveshr and @andreipaduroiu, but from an usability perspective, I don't know if doubling the number of configuration files is going to make things better for users.

Reply:
At the outset, I think its a good idea to have a single config file. I've created a separate issue #4843 for this item, as I don't think its in the scope of this PR.